main.go

```diff
@@ -504,18 +505,22 @@ func syncEnterpriseRepos(conf *config, gpr *github.PullRequestEvent) error {
 	// don't include pull request events with the "synchronize" action.
 	if gpr.GetAction() == "closed" && pr.GetMerged() {
 
-		// Only sync on Merges to master
+		// Only sync on Merges to master or release branches and
+		// verify release branches through [0-9].[0-9].x regex
 		branch := pr.GetBase()
 		if branch == nil {
 			return fmt.Errorf("syncEnterpriseRepos: Failed to get the base-branch of the PR: %v", branch)
 		}
+
+		syncBranches := regexp.MustCompile("(master|[0-9].[0-9].x)")
```

Comment:
Two comments:
- This will not cover Mender release `2.12.x`
- It could match a theoretical "feature" branch like "port-stuff-to-1.2.x-release", or "bla-blabla-master"

To fix both, I suggest something like `^(master|[0-9]+\.[0-9]+\.x)$`

Reply:
True xD 

But no need to review this atm. I'll ping you when this is ready to merge once again.

Right now I'm having some issues with accessing the private repos through gallienii, so this will have to wait anywho. 

I'm simply curling Gallieni atm